src/QCheck.mli

```diff
@@ -138,48 +138,48 @@ val assume_fail : unit -> 'a
 (** {2 Generate Random Values} *)
 module Gen : sig
   type 'a t = Random.State.t -> 'a
-  (** A random generator for values of type 'a *)
+  (** A random generator for values of type 'a. *)
 
   type 'a sized = int -> Random.State.t -> 'a
-  (** Random generator with a size bound *)
+  (** Random generator with a size bound. *)
 
   val return : 'a -> 'a t
-  (** Create a constant generator  *)
+  (** Create a constant generator. *)
 
   val (>>=) : 'a t -> ('a -> 'b t) -> 'b t
   (** Monadic bind for writing dependent generators. First generates an ['a] and then
       passes it to the given function, to generate a ['b]. *)
 
   val (<*>) : ('a -> 'b) t -> 'a t -> 'b t
   (** Infix operator for composing a function generator and an argument generator
-      into a result generator *)
+      into a result generator. *)
 
   val map : ('a -> 'b) -> 'a t -> 'b t
-  (** [map f g] transforms a generator [g] by applying [f] to each generated element *)
+  (** [map f g] transforms a generator [g] by applying [f] to each generated element. *)
 
   val map2 : ('a -> 'b -> 'c) -> 'a t -> 'b t -> 'c t
-  (** [map f g1 g2] transforms two generators [g1] and [g2] by applying [f] to each
-      pair of generated elements *)
+  (** [map2 f g1 g2] transforms two generators [g1] and [g2] by applying [f] to each
+      pair of generated elements. *)
 
   val map3 : ('a -> 'b -> 'c -> 'd) -> 'a t -> 'b t -> 'c t -> 'd t
-  (** [map f g1 g2 g3] transforms two generators [g1], [g2], and [g3] by applying [f]
-      to each triple of generated elements *)
+  (** [map3 f g1 g2 g3] transforms two generators [g1], [g2], and [g3] by applying [f]
```

Comment:
Since you're started on typos, this should probably read "transforms *three* generators"